app/src/main/res/drawable/continent_africa.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:viewportWidth="538.6851"
+    android:viewportHeight="645.0074"
+    android:width="538.68512dp"
+    android:height="645.00739dp">

```

Comment:
These should be `64dp` if I remember correctly. Bonus points for using `64px` in inkscape already and centering the drawing in it.

Reply:
It wasn't necessary because I set the width in the list item definition. But I've changed it, what did I win with the bonus points? 😆